tdrs-backend/tdpservice/users/permissions.py

```diff
@@ -233,4 +238,6 @@ def has_object_permission(self, request, view, obj):
             )
             return user_region == obj.stt.region_id
 
-        return obj == request.user or has_model_permission
+        # Check if user is an admin
+        is_admin = request.user.groups.filter(name__in=["OFA System Admin", "OFA Admin"]).exists()
```

Comment:
I think we should have one source of truth for admin users. The Group model has the Group names, we should be able to have pk available and use that instead of searching for group names

Reply:
Im open to this, but I dont know the historical/functional reasons why there are two admin groups. Maybe someone could illuminate this?

Reply:
What I was trying to say is instead of searching by admin names, we should filter by a pk

Reply:
> Im open to this, but I dont know the historical/functional reasons why there are two admin groups. Maybe someone could illuminate this?

The OFA Admin group is intended for a frontend 'admin UI' epic in the backlog and distinct from the technical system admin group. In general, the regular admin will not be involved in the same procedures that the sysadmin would.